### PR TITLE
fix: compatibility with upcoming changes

### PR DESCRIPTION
**Pull request by sidnov**
Fixes #[issue_no]

## Summary of Changes

**This build is NOT meant for general testing.** 

Any discussions/bug reports related to this build should be restricted to MSFS Forums, as per the NDA, unless stated otherwise:
https://forums.flightsimulator.com/t/fbw-a32nx-ctd-latest-dev-build/414893/129

## Additional context

Discord username (if different from GitHub):

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml [hunk 1]
```diff
@@ -1487,25 +1487,26 @@
...
-                    <WWISE_EVENT>lswitch</WWISE_EVENT>
-                </UseTemplate>
+                <Component ID="SWITCH_OVHD_INTLT_DOME" Node="SWITCH_OVHD_INTLT_DOME">
+                    <UseTemplate Name="ASOBO_GT_Switch_3States">
+                        <NODE_ID>SWITCH_OVHD_INTLT_DOME</NODE_ID>
+                        <PART_ID>SWITCH_OVHD_INTLT_DOME</PART_ID>
+                        <ANIM_NAME>SWITCH_OVHD_INTLT_DOME</ANIM_NAME>
+                        <WWISE_EVENT>lswitch</WWISE_EVENT>
+                        <DIMMER_POS_0_VALUE>100</DIMMER_POS_0_VALUE>
+				        <DIMMER_POS_1_VALUE>50</DIMMER_POS_1_VALUE>
```
Comment (Benjozork) [inline flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml:1497]: 🤔

Comment (Benjozork) [reply flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml:1497]: (formatting)

Comment (sidnov) [reply flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml]: ✅

File: flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml [hunk 2]
```diff
@@ -3815,7 +3816,7 @@
                 </RELEASE_EVENT>
                 <ALT_CODE>(L:A32NX_OVHD_INTLT_ANN, number) 2 == if{ 0.1 } els{ 1 } *</ALT_CODE>
                 <!-- As emissives aren't split between SEQ1 and SEQ2, we'll just require either bus to power both lights. -->
-                <EMISSIVE_CODE>(L:Generic_Master_Warning_Active) (L:A32NX_OVHD_INTLT_ANN) 0 == or (L:A32NX_ELEC_AC_ESS_SHED_BUS_IS_POWERED, Bool) (L:A32NX_ELEC_AC_2_BUS_IS_POWERED, Bool) or and</EMISSIVE_CODE>
+			    <IS_STATE_ACTIVE>(L:Generic_Master_Warning_Active) (L:A32NX_OVHD_INTLT_ANN) 0 == or (L:A32NX_ELEC_AC_ESS_SHED_BUS_IS_POWERED, Bool) (L:A32NX_ELEC_AC_2_BUS_IS_POWERED, Bool) or and</IS_STATE_ACTIVE>
```
Comment (tracernz) [inline flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml:3820]: Replace \t with sp (same for rest of commit).

Comment (sidnov) [reply flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml]: ✅

**Review by Benjozork — APPROVED** [review]

**Pull request merged**